src/pretix/control/templates/pretixcontrol/checkin/index.html

```diff
@@ -185,6 +185,7 @@ <h3 class="panel-title">{% trans "Filter" %}</h3>
                                             <span class="fa fa-magic text-muted"
                                                   data-toggle="tooltip" title="{% trans "Checked in automatically" %}"></span>
                                         {% endif %}
+                                        {% include "pretixcontrol/checkin/fragment_checkin_source_type.html" with checkin=e.checkins.last %}
```

Comment:
I don't think it's a good idea to make a database query for every line we show on the page, this would need to be prefetched in some way.